cli/src/main/kotlin/tools/samt/cli/CliArgs.kt

```diff
@@ -12,6 +12,9 @@ class CliArgs {
     @Parameter(names = ["--benchmark-runs"], description = "Parse files multiple times and measure execution time", help = true)
     var benchmarkRuns: Int = 100
 
+    @Parameter(names = ["--dumpast"], description = "Dump a visual representation of the AST", help = true)
```

Comment:
What about `--ast` instead?

Reply:
Fair point, I just somehow don't like the current name 🤔 
`--dump-ast` is more readable but a bit long, but if we want a `--dump-tokens`, `--dump-semantic` or `--dump-types` in the future this might make more sense

Reply:
That was my reasoning as well, once we add more flags to dump more things we might want to be clear about what is actually happening, instead of just "--semantic", which could mean any different amount of things.

Reply:
I agree, I will change it.